controllers/metal3.io/hostfirmwaresettings_controller.go

```diff
@@ -43,7 +43,7 @@ import (
 const (
 	provisionerRetryDelay          = time.Second * 30
 	resourceNotAvailableRetryDelay = time.Second * 30
-	reconcilerRequeueDelay         = time.Minute * 5
+	reconcilerRequeueDelay         = time.Minute * 1
```

Comment:
Maybe we should consider reducing this only when FirmwareSettingsChangeDetected is true?

Reply:
That's a really good idea. Changed.